framework-core/src/main/java/org/daisy/pipeline/job/Job.java

```diff
@@ -87,8 +88,8 @@ public final void initialise(){
                 DONE,
                 /** The ERROR. */
                 ERROR,
-                /** The VALIDATION_FAIL */
-                VALIDATION_FAIL
+                /** The FAIL */
+                FAIL
```

Comment:
I like this change, makes sense. However, for consistency, should `DONE` be renamed to `SUCCESS` now?

Reply:
Yes, perhaps. Good point.